src/Messages/Messages.js

```diff
@@ -137,7 +134,10 @@ export function Messages({ fromUserId }) {
               </svg>
             </div>
             <div className={styles.messagesSection}>
-              <div className={styles.messagesBody}>
+              <div
+                className={styles.messagesBody}
+                ref={(div) => div && (div.scrollTop = 100000)}
```

Comment:
Can you create an issue for this so we don't forget? 

"Clean up scrolling logic for messages container div".

Because it scrolls down whenever you re-render. To see the bug, scroll up, then type anything in the field, it will scroll down.

Reply:
Created https://github.com/alshakerM/my-instagram/issues/53.